locale/nb_NO/LC_MESSAGES/django.po

```diff
@@ -43,311 +43,501 @@ msgstr "norlønn-rapport"
+#: templates/base-navigation.jinja:41
 msgid "Invoicing"
 msgstr "Fakturering"
 
-#: templates/base.jinja:59
-msgid "Wages"
-msgstr "Lønn"
+#: templates/base-navigation.jinja:45
+msgid "Wage checklist and wage system export"
+msgstr ""
+
+#: templates/base-navigation.jinja:55
+#, fuzzy
+msgid "Go to administration panel"
+msgstr "Administrasjon"
 
-#: templates/base.jinja:67
+#: templates/base-navigation.jinja:57
 msgid "Administration"
 msgstr "Administrasjon"
 
-#: templates/base.jinja:72
+#: templates/base-navigation.jinja:62
 msgid "Log out of SPFs SPBM"
 msgstr "Logg ut av SPFs SPBM"
 
-#: templates/base.jinja:73
+#: templates/base-navigation.jinja:64
 msgid "Logout"
 msgstr "Logg ut"
 
-#: templates/index.jinja:2
-msgid "Society overview"
-msgstr "Foreningsoversikt"
+#: templates/base.jinja:47
+msgid "<strong>Empty page!</strong> Please contact the mailing list, as this page is incorrectly empty."
+msgstr "<strong>Tom side!</strong> Vennligst kontakt epostlisten."
 
-#: templates/index.jinja:5
-msgid "You're logged into an account associated with:"
-msgstr "Du er logget på en konto assosiert med:"
+#: templates/base.jinja:57
+msgid "Go to our source code repository"
+msgstr "Gå til kildekoden"
 
-#: templates/index.jinja:6
-msgid "As of right now, this page exactly doesn't really do anything, so take a look at the navigation bar!."
+#: templates/base.jinja:58
+msgid "Source code"
+msgstr "Kildekode"
+
+#: templates/base.jinja:61
+msgid "Go to our public contact page"
+msgstr "Gå til vårt offentlige kontaktpunkt"
+
+#: templates/base.jinja:62
+msgid "Knowledge base"
+msgstr "Kunnskapsbase"
+
+#: templates/base.jinja:64
+msgid "E-mail the board for wage issues and similarly"
+msgstr "Send epost til styret angående lønningsproblemer eller liknende."
+
+#: templates/base.jinja:65
+msgid "Board mailing list"
+msgstr "Styrets epostliste"
+
+#: templates/base.jinja:69
+msgid "For technical issues, limitations, feedback or other suggestions, please go to our issue tracker. For wage, worker or invoicing related issues, please contact the mailing list."
 msgstr ""
+"Ved tekniske problemer eller begresnsninger, eller dersom du har andre tilbakemeldinger, vennligst gå til vår issue tracker. For problemer relatert til lønning, arbeidere eller faktura, vennligst "
+"kontakt epostlisten."
+
+#: templates/delete.jinja:3
+#, fuzzy, python-format
+msgid "Confirm deletion of %(type)s"
+msgstr "Bekreft sletting av arrangement"
+
+#: templates/delete.jinja:11
+msgid "Confirm deletion"
+msgstr "Bekreft sletting"
+
+#: templates/delete.jinja:15
+#, fuzzy, python-format
+msgid "Are you sure that you wish to delete the %(type)s %(instance)s?"
+msgstr "Er du helt sikker på at du ønsker å slette arrangementet %(event)s?"
+
+#: templates/delete.jinja:20
+msgid "Cancel deleting and take me back"
+msgstr "Avbryt sletting og ta meg tilbake"
+
+#: templates/delete.jinja:22
+#, fuzzy, python-format
+msgid "Delete the %(type)s"
+msgstr "Slett arrangementet"
```

Comment:
Ooo, må bruke `%(type)s`, hvis ikke vil ikke slettemalen kunne brukes for andre objekter. Oversetting av objektnavn er avhengig av hvordan `models.py`-navn er oversatt.